lib/jsdom/living/helpers/colors.js

```diff
@@ -1,26 +1,158 @@
+  lightcyan: [0xE0, 0xFF, 0xFF],
+  lightgoldenrodyellow: [0xFA, 0xFA, 0xD2],
+  lightgray: [0xD3, 0xD3, 0xD3],
+  lightgreen: [0x90, 0xEE, 0x90],
+  lightgrey: [0xD3, 0xD3, 0xD3],
+  lightpink: [0xFF, 0xB6, 0xC1],
+  lightsalmon: [0xFF, 0xA0, 0x7A],
+  lightseagreen: [0x20, 0xB2, 0xAA],
+  lightskyblue: [0x87, 0xCE, 0xFA],
+  lightslategray: [0x77, 0x88, 0x99],
+  lightslategrey: [0x77, 0x88, 0x99],
+  lightsteelblue: [0xB0, 0xC4, 0xDE],
+  lightyellow: [0xFF, 0xFF, 0xE0],
   lime: [0x00, 0xFF, 0x00],
-  olive: [0x80, 0x80, 0x00],
-  yellow: [0xFF, 0xFF, 0x00],
+  limegreen: [0x32, 0xCD, 0x32],
+  linen: [0xFA, 0xF0, 0xE6],
+  magenta: [0xFF, 0x00, 0xFF],
+  maroon: [0x80, 0x00, 0x00],
+  mediumaquamarine: [0x66, 0xCD, 0xAA],
+  mediumblue: [0x00, 0x00, 0xCD],
+  mediumorchid: [0xBA, 0x55, 0xD3],
+  mediumpurple: [0x93, 0x70, 0xDB],
+  mediumseagreen: [0x3C, 0xB3, 0x71],
+  mediumslateblue: [0x7B, 0x68, 0xEE],
+  mediumspringgreen: [0x00, 0xFA, 0x9A],
+  mediumturquoise: [0x48, 0xD1, 0xCC],
+  mediumvioletred: [0xC7, 0x15, 0x85],
+  midnightblue: [0x19, 0x19, 0x70],
+  mintcream: [0xF5, 0xFF, 0xFA],
+  mistyrose: [0xFF, 0xE4, 0xE1],
+  moccasin: [0xFF, 0xE4, 0xB5],
+  navajowhite: [0xFF, 0xDE, 0xAD],
   navy: [0x00, 0x00, 0x80],
-  blue: [0x00, 0x00, 0xFF],
+  oldlace: [0xFD, 0xF5, 0xE6],
+  olive: [0x80, 0x80, 0x00],
+  olivedrab: [0x6B, 0x8E, 0x23],
+  orange: [0xFF, 0xA5, 0x00],
+  orangered: [0xFF, 0x45, 0x00],
+  orchid: [0xDA, 0x70, 0xD6],
+  palegoldenrod: [0xEE, 0xE8, 0xAA],
+  palegreen: [0x98, 0xFB, 0x98],
+  paleturquoise: [0xAF, 0xEE, 0xEE],
+  palevioletred: [0xDB, 0x70, 0x93],
+  papayawhip: [0xFF, 0xEF, 0xD5],
+  peachpuff: [0xFF, 0xDA, 0xB9],
+  peru: [0xCD, 0x85, 0x3F],
+  pink: [0xFF, 0xC0, 0xCB],
+  plum: [0xDD, 0xA0, 0xDD],
+  powderblue: [0xB0, 0xE0, 0xE6],
+  purple: [0x80, 0x00, 0x80],
+  rebeccapurple: [0x66, 0x33, 0x99],
+  red: [0xFF, 0x00, 0x00],
+  rosybrown: [0xBC, 0x8F, 0x8F],
+  royalblue: [0x41, 0x69, 0xE1],
+  saddlebrown: [0x8B, 0x45, 0x13],
+  salmon: [0xFA, 0x80, 0x72],
+  sandybrown: [0xF4, 0xA4, 0x60],
+  seagreen: [0x2E, 0x8B, 0x57],
+  seashell: [0xFF, 0xF5, 0xEE],
+  sienna: [0xA0, 0x52, 0x2D],
+  silver: [0xC0, 0xC0, 0xC0],
+  skyblue: [0x87, 0xCE, 0xEB],
+  slateblue: [0x6A, 0x5A, 0xCD],
+  slategray: [0x70, 0x80, 0x90],
+  slategrey: [0x70, 0x80, 0x90],
+  snow: [0xFF, 0xFA, 0xFA],
+  springgreen: [0x00, 0xFF, 0x7F],
+  steelblue: [0x46, 0x82, 0xB4],
+  tan: [0xD2, 0xB4, 0x8C],
   teal: [0x00, 0x80, 0x80],
-  aqua: [0x00, 0xFF, 0xFF]
+  thistle: [0xD8, 0xBF, 0xD8],
+  tomato: [0xFF, 0x63, 0x47],
+  turquoise: [0x40, 0xE0, 0xD0],
+  violet: [0xEE, 0x82, 0xEE],
+  wheat: [0xF5, 0xDE, 0xB3],
+  white: [0xFF, 0xFF, 0xFF],
+  whitesmoke: [0xF5, 0xF5, 0xF5],
+  yellow: [0xFF, 0xFF, 0x00],
+  yellowgreen: [0x9A, 0xCD, 0x32]
 };
 
 function colorKeywordToRGB(name) {
-  if (name.toLowerCase() in RGB_LOOKUP) {
+  if (Object.prototype.hasOwnProperty.call(RGB_LOOKUP, name.toLowerCase())) {
```

Comment:
Just a suggestion from a newbie here, but instead of a `Object.prototype.hasOwnProperty.call`, would a simple object access that gives `undefined` be less scarry and  in this case?
*I put a ternary here, but keeping the if can be more readable too.
```js
function colorKeywordToRGB(name) {
  const rgbKey = name.toLowerCase();

  return RGB_LOOKUP[rgbKey] ? `rgb(${RGB_LOOKUP[rgbKey].join(", ")})` : "";
}
```